data/seeds/007_messages.js

```diff
@@ -28,7 +28,7 @@ exports.seed = function (knex) {
     },
     {
       title: 'Where is my achievement?',
-      message: "my achievement didn't pop up when I did class.",
+      message: "my achievement didn't pop up when I did course.",
```

Comment:
same deal as previous comment - matter of spoken word awkwardness.
```suggestion
      message: "my achievement didn't pop up when I completed class.",
```

Reply:
These are seeds for messages between users, so I believe it's fine if the language used isn't exactly proper.